examples/tracking-mask/rs-tracking-mask.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+#include <librealsense2/rs.hpp>
+#include <librealsense2/rsutil.h>
+#include <array>
+#include <cmath>
+#include <iostream>
+#include <vector>
+#include "example.hpp"
+#include <string.h> // memset
+#include <memory>
+
+std::unique_ptr<uint8_t[]> mask_image(rs2::video_frame & frame, uint8_t * mask)
+{
+    uint8_t * frame_data = (uint8_t *) frame.get_data();
+    int width = frame.get_width();
+    int height = frame.get_height();
+    int mask_width = width / 8;
+    std::unique_ptr<uint8_t []> output(new uint8_t[width*height]);
+    // Iterate over each pixel in the image and check the mask to see
+    // if the pixel is visible to the tracking algorithm or not.
+    for(int y = 0; y < height; y++) {
+        int my = y / 8;
+        for(int x = 0; x < width; x++) {
+            int mx = x / 8;
+            if(mask[my*mask_width + mx])
+                output[y * width + x] = frame_data[y * width + x];
+            else
+                output[y * width + x] = 0;
+        }
+    }
+    return output;
+}
+
+int main(int argc, char * argv[]) try
+{
+    std::cout << "Waiting for device..." << std::endl;
+
+    // Declare RealSense pipeline, encapsulating the actual device and sensors
+    rs2::pipeline pipe;
+    // Create a configuration for configuring the pipeline with a non default profile
+    rs2::config cfg;
+    // Enable fisheye streams
+    cfg.enable_stream(RS2_STREAM_FISHEYE, 1);
+    cfg.enable_stream(RS2_STREAM_FISHEYE, 2);
+    // Start pipeline with chosen configuration
+    rs2::pipeline_profile pipe_profile = pipe.start(cfg);
+
+    // T265 has two fisheye sensors, we can choose any of them (index 1 or 2)
+    const int fisheye_sensor_idx = 1;
+
+    // Get fisheye sensor intrinsics parameters
+    rs2::stream_profile fisheye_stream = pipe_profile.get_stream(RS2_STREAM_FISHEYE, fisheye_sensor_idx);
+    rs2_intrinsics intrinsics = fisheye_stream.as<rs2::video_stream_profile>().get_intrinsics();
+    auto tm2 = pipe_profile.get_device().as<rs2::tm2>();
+
+    // Create an OpenGL display window and a texture to draw the masked fisheye image
+    window app(intrinsics.width, intrinsics.height, "Intel RealSense T265 Tracking Mask Example");
+    window_key_listener key_watcher(app);
+    texture fisheye_image;
+
+    bool mask_set = false;
+    // Main loop
+    while (app)
+    {
+        {
+            // Wait for the next set of frames from the camera
+            auto frames = pipe.wait_for_frames();
+            // Get a frame from the fisheye stream
+            rs2::video_frame fisheye_frame = frames.get_fisheye_frame(fisheye_sensor_idx);
+
+            if(!mask_set) {
+                int width = intrinsics.width/8;
+                int height = intrinsics.height/8;
+                auto mask_ptr = std::unique_ptr<uint8_t []>(new uint8_t[width*height]);
+                double global_ts_ms = fisheye_frame.get_timestamp();
```

Comment:
This is a potential pitfall - please highlight the significance of setting this param properly

Reply:
Done in readme and also mentioned in api doc